Dockerfile

```diff
@@ -36,6 +37,14 @@ RUN git clone --recursive https://github.com/jupp0r/prometheus-cpp/ && \
 	make -j2 && \
 	make install
 
+RUN git clone --depth=1 https://github.com/minetest/irrlicht/ -b ${IRRLICHT_VERSION} && \
+	mkdir irrlicht/build && \
```

Comment:
`cp -r irrlicht/include /usr/include/irrlichtmt` would be enough here, the server doesn't need irrlicht built
(you can drop bzip2-dev, libpng-dev, jpeg-dev, libxxf86vm-dev, mesa-dev, libogg-dev, libvorbis-dev, openal-soft-dev, freetype-dev then too)

Reply:
i'll try to drop some libs, you are right :D i used the original method at the beginning of this dockerfile. That should massiverly lightweight the image